src/constants/translations/en/accordion.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Accordion is a component, translations should be moved to guestHomePage translation file. And there create your block and place translations. And do not use heading and subheading, its title and description